### PR TITLE
[WIP] CONSOLE-4083: Remove Dockerfile.builder and make other container file changes

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: .ci-operator.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 build_root_image:
+  # what do do here?
```
Comment (rhamilto) [inline .ci-operator.yaml:2]: This file will be removed after the initial changes included in this PR.

File: Dockerfile.cypress [hunk 2]
```diff
@@ -0,0 +1,68 @@
+##################################################
+#
+# cypress image
+FROM cypress/included:12.17.4
```
Comment (rhamilto) [inline Dockerfile.cypress:4]: This will need to be updated once https://github.com/openshift/console/pull/13887 merges

File: Dockerfile.cypress [hunk 3]
```diff
@@ -0,0 +1,68 @@
+##################################################
+#
+# cypress image
+FROM cypress/included:13.10.0
+
+##################################################
+#
+# go backend build
```
Comment (jhadvig) [inline Dockerfile.cypress:8]: Thinking about it, we wont need to build backend in the this image, since we are testing agains live cluster.

**Pull request closed without merging**